styles/text-utility/text-utility.css

```diff
@@ -0,0 +1,71 @@
+:root {
+    --color-text: #f97316;
+}
+
+.grid-two {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 2rem;
+}
+
+.h1 {
+    font-size: 48px;
+    line-height: 64px;
```

Comment:
Try to use rem for values of 10px and more

Reply:
will change that for sure
thanks